wire/wire.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package basictracer_go.wire;
```

Comment:
agree that the proto should move outside of the main repo. I'd do that rather sooner than later so that the other impls can catch up.